CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [?.?.?] - ????-??-??
+### Fixed
```

Comment:
This seems like a `6.4.3` to me.

Reply:
Sounds good.

Reply:
Not sure about the workflow for releasing, but I went ahead and changed the date to today and put 6.4.3 in the version number here. Hope that's right.